### PR TITLE
Fix yq image for aws-efs-operator

### PR DESCRIPTION
**Pull request by cblecker**

### CODE REVIEW

**Comment by cblecker** [comment]
/hold
/assign @2uasimojo @dustman9000 
FYI @jharrington22 

This operator doesn't appear to be transitioned over to app-interface. Can you have a look?

File: hack/app_sre_create_image_catalog.sh [hunk 1]
```diff
@@ -2,7 +2,9 @@
 
 # Boilerplate: change this, then you shouldn't have to muck with the
 # rest of the file.
-OPERATOR_NAME=aws-efs-operator
+# prefix var with _ so we don't clober the var used during the Make build
```
Comment (2uasimojo) [inline hack/app_sre_create_image_catalog.sh:5]:
```suggestion
# prefix var with _ so we don't clobber the var used during the Make build
```

**Review by 2uasimojo — APPROVED** [review]
This script is unused, since we're not deploying this operator via app-interface. So no harm in landing this -- and maybe it keeps this repo off the radar when we're looking for this problem in the future.

/lgtm
/approve
/hold cancel

**Comment by 2uasimojo** [comment]
> This operator doesn't appear to be transitioned over to app-interface.

It's a [community operator](https://github.com/operator-framework/community-operators/tree/master/community-operators/aws-efs-operator).

**Pull request merged**